pkg/network/dhcp/server/server.go

```diff
@@ -156,7 +158,7 @@ func prepareDHCPOptions(
 			ntpServers := [][]byte{}
 
 			for _, server := range customDHCPOptions.NTPServers {
-				ip := net.ParseIP(server).To4()
+				ip := netutils.ParseIPSloppy(server).To4()
```

Comment:
I think that changing this here may cause a maintenance problem in the future.
It will be much better to normalize/adjust the values as soon as possible in the setup flow. Then, if any changes will get introduced later on (refactoring or others), it will still work as expected.

If I am not mistaken, this code is in the setup flow, so we should able to cover it early on (e.g. at `netconf`).
Can you please explore this option?

Reply:
Sure.